event_contact/tests/test_event.py

```diff
@@ -2,13 +2,13 @@
 # Copyright 2019 Alexandre Díaz <alexandre.diaz@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp.tests.common import TransactionCase
+from odoo.tests.common import TransactionCase
 
 
 class EventCase(TransactionCase):
-    def setUp(self, *args, **kwargs):
-        super().setUp(*args, **kwargs)
-
+    @classmethod
+    def setUpClass(self, *args, **kwargs):
+        super().setUpClass(*args, **kwargs)
```

Comment:
By convention `@classmethod` methods use `cls` instead of `self` as the first parameter receives the class itself, not an instance of the class

Aside from that, the regular `setUpClass` method for `TransactionCase` doesn't have any extra arguments

```suggestion
    def setUpClass(cls):
        super().setUpClass()
```